### PR TITLE
add break after key is found in for loop

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca runcom@linux.com

### CODE REVIEW

**Comment by duglin** [comment]
LGTM

File: runconfig/merge.go [hunk 1]
```diff
@@ -33,6 +33,7 @@ func Merge(userConf, imageConf *Config) error {
 				userEnvKey := strings.Split(userEnv, "=")[0]
 				if imageEnvKey == userEnvKey {
 					found = true
+					break
```
Comment (LK4D4) [inline runconfig/merge.go]: then `found` variable isn't needed at all

Comment (runcom) [inline runconfig/merge.go]: yeah right

Comment (runcom) [inline runconfig/merge.go]: mm how the `!found` check below is being executed then?

Comment (LK4D4) [inline runconfig/merge.go]: just remove `if !found`, if you reached that code, then you're not breaked - `!found`

Comment (runcom) [inline runconfig/merge.go]: but without the check the outer for will just append the env :/

Comment (LK4D4) [inline runconfig/merge.go]: Ah, okay, you're right, sorry.

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**